src/appleseedmaya/exporters/cameraexporter.cpp

```diff
@@ -161,11 +166,32 @@ void CameraExporter::createEntities(
         cameraParams.insert("shift_y", shift_y);
 
         // Maya's focal_length is given in mm so we convert it to meters.
-        cameraParams.insert("focal_length", cameraFn.focalLength() * 0.001);
+        cameraParams.insert("focal_length", cameraFn.focalLength() / cameraFn.cameraScale() * 0.001);
+        cameraParams.insert("horizontal_fov", asf::rad_to_deg(cameraFn.horizontalFieldOfView() * cameraFn.cameraScale()));
 
+        // Physical camera (thin-lens camera) parameter.
         if (dofEnabled)
         {
-            // TODO: extract dof params here...
+            bool autofocusEnabled = false;
+            if (AttributeUtils::get(node(), "asAutofocus", autofocusEnabled))
+            {
+                bool useAutofocus = autofocusEnabled;
```

Comment:
You don't need this variable, just use `autofocusEnabled`.

It would also be cleaner to skip this block entirely if autofocus is disabled, no? Something like:

    bool autofocusEnabled = false;
    if (AttributeUtils::get(node(), "asAutofocus", autofocusEnabled) && autofocusEnabled)
    {
        ...
    }